README.md

```diff
@@ -1,122 +1,241 @@
-        --insecure          Whether or not localhost connections and self-signed certificates are allowed
-    -q, --quiet             Quiet mode (no warnings or informative messages, only errors)
-    -V, --version           Prints version information
-    -v, --verbose           Verbose mode (-v, -vv, -vvv, etc.)
+    -h, --help             Prints help information
+        --insecure         Whether or not localhost connections and self-signed certificates are allowed
+    -q, --quiet            Quiet mode (no warnings or informative messages, only errors)
+        --rsync-disable    Disable writing the rsync files
+    -V, --version          Prints version information
+    -v, --verbose          Verbose mode (-v, -vv, -vvv, etc.)
 
 OPTIONS:
-        --cleanup-after <seconds>    The minimum number of seconds that a dangling snapshot or delta must have been
-                                     published by krill-sync before it can be removed [default: 600]
-    -f, --format <format>            Output both RRDP and Rsync style repositories or only one of them? [default: BOTH]
-                                     [possible values: BOTH, RRDP, RSYNC]
-        --pid-file <pid-file>        The location to write our process ID to [default: /var/run/krill-sync.pid]
-        --rrdp-dir <rrdp-dir>        The directory to write RRDP files to [default: /var/lib/krill-sync/rrdp]
-        --rsync-dir <rsync-dir>      The directory to write Rsync files to [default: /var/lib/krill-sync/rsync]
-    -s, --state-dir <state-dir>      The directory to write state to [default: /var/lib/krill-sync]
+        --cleanup-after <seconds>        Remove unreferenced files and directories older than X seconds [default: 600]
+        --pid-file <file>                The location to write our process ID to [default: /var/run/krill-sync.pid]
+        --rrdp-dir <dir>                 The directory to write RRDP files to [default: /var/lib/krill-sync/rrdp]
+        --rrdp-max-deltas <number>       Optional hard upper limit to the number of deltas
+        --rrdp-notify-delay <seconds>    Delay seconds before writing the notification.xml file [default: 0]
+        --rsync-dir <dir>                The directory to write Rsync files to [default: /var/lib/krill-sync/rsync]
+        --source_uri_base <uri>          Base uri for the notify file on the back-end server. Must end with a slash
+    -s, --state-dir <dir>                The directory to write state to [default: /var/lib/krill-sync]
 
 ARGS:
-    <notification-uri>    The RRDP notification file URI of the Krill instance to sync with
+    <notification-uri>    The public RRDP notification URI
 ```
 
 _**Tip:** Unlike the `-V` argument, the `--version` argument also prints out the Git commit from which krill-sync was built._
 
 _**Tip:** If connecting directly to a Krill server, and not to e.g. NGINX in front of Krill, you will need to use `--insecure` as the Krill RRDP service uses a self-signed TLS certificate._
```

Comment:
Krill specific. Remove this tip?

Reply:
No, I believe this is fine. There are krill specific users and others will be able to understand that if they use self-signed certificates they need to do the same.